### DISCUSSION TITLE
Tracking multiple files

### DISCUSSION THREAD

**Discussion by RionIwasaki** [WeirongChen/DeepEdge#6] [Q&A]
Hi Weirong,
Have a question about tracking multiple files. I know there is an option to track across all files in a file list. I wonder if the setting for cropping is the same across these files. For example, you have five files. You crop the first file before tracking. Are the subsequent four files cropped in the same way as the first one? 

Relatedly, I have five files (each block) from one experiment. A participant didn't remove the stabilization helmet during the experiment. However, the fit naturally changed ever slightly from block to block. Therefore, the area of interest after cropping that is optimal for tracking could change from file to file. In this case, should I change the area of cropping from file to file?

**Comment by WeirongChen** [comment]
Thanks for the questions Rion!

1. Are the subsequent four files cropped in the same way as the first one?
> Yes, the subsequent file will be cropped in the same setting if you don't change it.  

2. The idea of cropping is to exclude non-ultrasound image area (e.g., meta information like name of clinician, date of experiment, name of the patient).  The cropping area is NOT the area of interest and it should include the whole fan-shaped ultrasound imaging and should not change due to the change of position of the probe stabilizer.

**Reply by RionIwasaki** [reply]
Thank you so much for the answer, Weirong! 
_> Yes, the subsequent file will be cropped in the same setting if you don't change it.__
I see. 
_>The idea of cropping is to exclude non-ultrasound image area...The cropping area is NOT the area of interest._
The non-ultrasound image area! Yes, that's how I wanted to phrase my question. I was not able to come up with a better way to phrase it, so I used the area of interest instead, which is obviously not correct. 
_>it should include the whole fan-shaped ultrasound imaging and should not change due to the change of position of the probe stabilizer._
So, in my case where I have five block files from one experiment, I can just track all files at one. Thank you.
